### DISCUSSION TITLE
The system cannot find the path specified

### DISCUSSION THREAD

**Discussion by shibjeedanuwar** [norkunas/youtube-dl-php#199] [Q&A]
```php
<?php

require_once 'vendor/autoload.php';
// require_once'vendor/autoload.php';

$yt = new \YoutubeDl\YoutubeDl();
$yt->setBinPath('/usr/local/bin/yt-dlp');
$yt->debug(function ($type, $buffer) {
    if (\Symfony\Component\Process\Process::ERR === $type) {
        echo $buffer;
    } else {
        echo $buffer;
    }
});

$options = \YoutubeDl\Options::create()
    ->downloadPath('C:\Users\theop\Downloads')
    ->url('https://youtu.be/4A67iTiOD_g');

$collection = $yt->download($options);

// echo "Download completed\n";

?>
```
output:The system cannot find the path specified.

**Comment by norkunas** [comment]
As I'm not using windows so therefore cannot help properly, but googling this says:
> The Windows error “The system cannot find the specified path” usually occurs if you use an invalid path when downloading or saving, if corrupt system files are present or a directory has been deleted or changed.

**Comment by borneagle71** [comment]
Fairly certain that /usr/local/bin/yt-dlp wont exist on a windows machine, that's the directory that you'd use on a linux based system. When you download yt-dlp for windows you'll get an exe file and you'll need to set the bin path to wherever that exe file is.
